lib/rubocop/directive_comment.rb

```diff
@@ -42,7 +37,7 @@ def single_line?
 
     # Checks if this directive contains all the given cop names
     def match?(cop_names)
-      cops == cop_names.uniq.sort
+      parsed_cop_names&.uniq&.sort == cop_names.uniq.sort
```

Comment:
It'd be good if `parsed_cop_names` just returned `[]` instead of nil so that we wouldn't need this chain of `&.`

Reply:
thanks, fixed